addons/website/static/src/scss/website.backend.scss

```diff
@@ -86,38 +86,28 @@
     display: contents;
 }
 
-// Mobile preview
-%o-mobile-phone {
-    @include media-breakpoint-up(md) {
-        height: 735px !important;
-        width: 362px;
-        left: 50%;
-        top: 50%;
-        transform: translate(-50%, -50%);
-
-        // Below the following height, the preview size decreases to ensure it
-        // will always be fully visible.
-        @media(max-height: calc(800px + #{$o-navbar-height})) {
-            height: 602px !important;
-            width: 310px;
-
-            div.o_mobile_preview_layout, img.img_mobile {
-                height: 659px;
-                width: 342px;
-                top: -30px;
-                left: -17px;
-            }
-        }
+@mixin o-mobile-phone {
+    position: absolute;
+    left: 50%;
+    top: 50%;
+    width: 368px;
+    height: 763px;
+    transform: translate(-50%, -50%);
+    box-shadow: 0 4px 20px rgba(0, 0, 0, 0.15),
+                16px 12px 32px rgba(0, 0, 0, 0.15),
+                64px 0 64px rgba(0, 0, 0, 0.15);
+    border-radius: 2rem;
+
+    @media (max-height: 600px) {
+        transform: translate(-50%, -50%) scale(0.6);
+    }
 
-        .o_mobile_preview_layout, .img_mobile {
-            position: absolute;
-            pointer-events: none;
-            // The mobile phone image placed on top of the iframe must be moved
-            // to align the outlines of the iframe with the outlines of the
-            // mobile screen.
-            top: -36px;
-            left: -20px;
-        }
+    @media (min-height: 601px) and (max-height: 768px) {
+        transform: translate(-50%, -50%) scale(0.75);
+    }
+
+    @media (min-height: 769px) and (max-height: 900px) {
+        transform: translate(-50%, -50%) scale(0.85);
```

Comment:
This make the phone device too small on my screen (Philips 243B) compared to how it was before.

![image](https://github.com/user-attachments/assets/b7f9fe17-f04d-4e7e-a3b6-8b6ffcbdae85)

I think that using media queries to define the best height for the mobile device is not the right way to handle this. It should be possible to adapt the height of the device responsively by anchoring it to the top and bottom of the page (with margins, of course). This way, it will adjust dynamically to the window height, pixel by pixel, instead of changing at specific breakpoints. What do you think? Could you check if this is possible, please? 😊